Taskfile.yml

```
@@ -0,0 +1,200 @@
+    cmds:
+      - brew install uv pre-commit
+      - uv sync
+      - chmod +x app/main.sh
+      - chmod +x setup.sh
+      - chmod +x timer-examples.sh
+
+  crawler:
+    desc: Run GTFS RT crawler with specified interval
+    summary: |
+      Run GTFS Realtime crawler for specified interval.
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)